Sateltiles/admin_user.py

```diff
@@ -0,0 +1,151 @@
+        manage_button.Bind(wx.EVT_BUTTON, self.manageUser)
+
+        # database admin/information
+        database_name = "gdal2tiles"
+        database_username = "postgres"
+        database_password = "polpol01"
+        database_host = "203.159.29.196"
+        database_port = "5432"
+        try:
+            database = db.PostGresAPI(database_name, database_username, database_password, database_host, database_port) # connect to main database
+        except:
+            print("Can not connect to database (" + str(database_host) + ":" + str(database_port))
+
+        if role != "superadmin":  # only super admin can see this button
+            manage_button.Show(False)
+        
+        self.list = wx.ListCtrl(panel,pos=(20,0+50), size=(550,260), style=wx.LC_REPORT 
+                                 | wx.BORDER_SUNKEN
+                                 #| wx.BORDER_NONE         <<< This is up to your style uncomment # to enable
+                                 #| wx.LC_EDIT_LABELS
+                                 #| wx.LC_SORT_ASCENDING
+                                 #| wx.LC_NO_HEADER
+                                 #| wx.LC_VRULES
+                                 #| wx.LC_HRULES
+                                 #| wx.LC_SINGLE_SEL
+                                  )
+        
+        # Insert column
+        self.list.InsertColumn(0,"Name")
+        self.list.InsertColumn(1,"Min lat")
+        self.list.InsertColumn(2,"Max lat")
+        self.list.InsertColumn(3,"Min long")
+        self.list.InsertColumn(4,"Max long")
+        self.list.InsertColumn(5,"EPSG")
+        self.list.InsertColumn(6,"Create by")
+        self.list.InsertColumn(7,"Create on")
+        self.list.InsertColumn(8,"Hide")
+        
+        table = database.getTileData()
+
+        for row in range(len(table)):
+            pos = self.list.InsertStringItem(row, table[row][1])
+            self.list.SetStringItem(pos, 1, table[row][2])
+            self.list.SetStringItem(pos, 2, table[row][4])
+            self.list.SetStringItem(pos, 3, table[row][3])
+            self.list.SetStringItem(pos, 4, table[row][5])
+            self.list.SetStringItem(pos, 5, table[row][6])
+            self.list.SetStringItem(pos, 6, table[row][7])
+            self.list.SetStringItem(pos, 7, table[row][8])
+            self.list.SetStringItem(pos, 8, table[row][9])
+            
+
+        self.tif_path = ""
+        self.save_path = ""
+    
+        self.list.Show(True)
+        self.Show(True)
+
+    def backHome(self, evt):
+        self.Destroy() # close LoginPage
+        log.LoginPage(None, title="LoginPage")  # open up login page
+
+    def getTMS_Link(self, evt):
+        count = self.list.GetItemCount()
+        item_data = []
+        for row in range(count):
+            if self.list.IsSelected(row): # get selected item data
+                print ("You have selected item " + str(row+1))

```

Comment:
same here...again dont print in here...if you use this for debug purpose better remove unless it provides significant meaning